indexer/data_header.hpp

```diff
@@ -71,6 +72,7 @@ namespace feature
     /// @name Serialization
     //@{
     void Save(FileWriter & w) const;
+    void SaveRegionData(FileWriter & w) const;
```

Comment:
Also, consider to store region info directly in data header section, instead of an auxiliary one. Feel free to increase a data format version, if you need.